Dockerfile

```diff
@@ -18,7 +18,7 @@ ADD aws-opentelemetry-distro/ ./aws-opentelemetry-distro/
 # * https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/azure-functions/recover-python-functions.md#troubleshoot-cannot-import-cygrpc
 RUN sed -i "/opentelemetry-exporter-otlp-proto-grpc/d" ./aws-opentelemetry-distro/pyproject.toml
 
-RUN mkdir workspace && pip install --target workspace ./aws-opentelemetry-distro
+RUN mkdir workspace && pip install urllib3==2.2.3 --target workspace ./aws-opentelemetry-distro
```

Comment:
Pls add a comment to explain why we need to add the version restriction here. And note that we'll remove it after py 3.8 support deprecated